webpack/src/Components/EmptyState/EmptyStateActions.js

```diff
@@ -0,0 +1,20 @@
+import { API_OPERATIONS } from 'foremanReact/redux/API/APIConstants';
+import { ADD_CONTENT, FETCHING_KEY } from './Constants';
+
+export const AddEmptyStateContent = header => ({
+  type: ADD_CONTENT,
+  payload: header,
+});
+
+/*
+This action fetches data from the server
+For accessing the response's data, select it via this path: state.api.<FETCHING_KEY>.response`
+For further information please visit APIMiddleware page in foreman's storybook
+*/
+
+export const fetchData = url => ({
+  type: API_OPERATIONS.GET,
+  key: FETCHING_KEY, // you will need to re-use this key in order to access the right API reducer later.
+  url,
+  payload: {},
+});
```

Comment:
instead you can use the `get` helper from Foreman, e.g:
```js
import { get } from 'foremanReact/redux/API';
get({ key, url })
```

Reply:
Thanks, added